.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,89 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Thanks for taking the time to fill out this bug report!
+        ### The following information will help us resolve your issue.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to Reproduce
+      description: |
+        Please copy and paste the code you were trying to run that caused the error.
+
+        Feel free to include as little or as much as you think is relevant. This section will be automatically formatted into code, so no need for backticks.
+      render: python
```

Comment:
```suggestion
```

I don't like assuming this. I often want to add plain text here, or show code in other languages e.g. bash